gslib/addlhelp/command_opts.py

```diff
@@ -35,8 +35,13 @@
 
 
 <B>OPTIONS</B>
-  -D          Shows HTTP requests/headers and additional debug info needed when
-              posting support requests, including exception stack traces.
+  -D          Shows HTTP requests/headers and additional debug info needed
+              when posting support requests, including exception stack traces.
+
+              CAUTION: The output from using this flag includes authentication
```

Comment:
I was one the fence about that - on the one hand it seemed pretty clear (to me at least) that what applies to -D also applies to -DD, and I wanted to avoid crowding the page with cautions. On the other hand, we don't want to leave room for ambiguity.

Instead of duplicating the note, I'll try a different approach... let me know if you're not a fan.